programs/drift/src/controller/orders.rs

```diff
@@ -1664,6 +1679,19 @@ fn fulfill_perp_order(
             }
             PerpFulfillmentMethod::Match(maker_key, maker_order_index) => {
                 let mut maker = makers_and_referrer.get_ref_mut(maker_key)?;
+
+                // doesn't meet match condition for protected maker
+                if maker.is_protected_maker()
+                    && !user.can_skip_auction_duration(user_stats, now)?
```

Comment:
probably nicer if this goes into get_maker_orders_info since thats where we select the makers

Reply:
also this might be w jit